tests/datasource/data_connector/test_sql_data_connector.py

```diff
@@ -2475,16 +2475,20 @@ def test_introspect_db(
 
 
 @pytest.mark.integration
-def test_include_schema_name_introspection(mysql_sqlalchemy_datasource):
+def test_include_schema_name_introspection(mysql_engine):
+    execution_engine = SqlAlchemyExecutionEngine(
```

Comment:
Using an ExecutionEngine since this test is still relevant.

Reply:
Great